src/main/java/com/undertheriver/sgsg/common/domain/BaseEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.undertheriver.sgsg.common.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@MappedSuperclass
+@EntityListeners({AuditingEntityListener.class})
+public class BaseEntity {
+
+	@CreatedDate
+	private LocalDate createdAt;
+
+
+	@LastModifiedDate
+	private LocalDate updatedAt;
+
+	private boolean deleted = false;
```

Comment:
```suggestion
	private Boolean deleted;
```
으로 nullable하게 하는게 어떨까요? 엔티티가 생성될때마다 디폴트값이 있으면 생성될 때 락이 걸릴 것 같은데요.
만약 벌크인서트처럼 대용량의 생성있다면 위험할 것 같네요. null이거나 false일 때 active하다고 판단하면 좋을 것 가텐요.

Reply:
(https://americanopeople.tistory.com/292)[https://americanopeople.tistory.com/292] 자료에 락에 관련된 내용있으니 봐보셔도 좋을 것 같아요! 포스트그레스큐엘이지만 비슷할 것 같습니다.